heron/spi/src/java/com/twitter/heron/spi/utils/SchedulerUtils.java

```diff
@@ -43,41 +42,6 @@ private SchedulerUtils() {
   }
 
   /**
-   * Invoke the onScheduler() in IScheduler directly as a library
-   *
-   * @param config The Config to initialize IScheduler
-   * @param runtime The runtime Config to initialize IScheduler
-   * @param scheduler the IScheduler to invoke
-   * @param packing The PackingPlan to scheduler for OnSchedule()
-   * @return true if scheduling successfully
-   */
-  public static boolean onScheduleAsLibrary(
```

Comment:
`onSchedulerAsLibrary` is a method to launch the scheduler. A launcher performs this function. So I think this method belongs to `LauncherUtils`. Moreover most other methods in `SchedulerUtils` are used by a scheduler once launched, for e.g. set location etc.